lib/api/create_constraint.rb

```diff
@@ -1,7 +1,8 @@
 module Api
   class CreateConstraint
     def matches?(request)
-      Hash(JSON.parse(request.body.read)).fetch("action", "create").in?(%w(create add))
+      body = request.body.read
+      Hash(JSON.parse(body)).fetch("action", "create").in?(%w(create add)) unless body.empty?
```

Comment:
So, if  `body.empty?` is true, it will return nil?  Should we return false?

Reply:
@kbrock see above

Reply:
:+1: agreed, since it's a `?` method, let's return true / false